test_broken.py

```diff
@@ -9,27 +9,23 @@
 def test_extra_length(region_name='extra'):
     assert len(Database().get_extra()) == expected_len(region_name)
 
-
```

Comment:
Flake8 does not love the blank line counts https://travis-ci.org/LazoCoder/Pokemon-Terminal/builds/265651400#L453 but who cares...  we now have tons more Pokemon!